src/main/scala/it/pagopa/interop/agreementprocess/common/Adapters.scala

```diff
@@ -111,14 +122,21 @@ object Adapters {
   }
 
   implicit class DocumentWrapper(private val d: AgreementManagement.Document) extends AnyVal {
-    def toApi: Document = Document(
+    def toApi: Document                          = Document(
       id = d.id,
       name = d.name,
       prettyName = d.prettyName,
       contentType = d.contentType,
       createdAt = d.createdAt,
       path = d.path
     )
+    def toSeed: AgreementManagement.DocumentSeed = AgreementManagement.DocumentSeed(
+      id = d.id,
+      name = d.name,
+      prettyName = d.prettyName,
+      contentType = d.contentType,
+      path = d.path
+    )
```

Comment:
This is just to avoid sending to persistence something that already exists by forgetting to `copy`.

```suggestion
    def toSeed(newId: UUID, newPath: String): AgreementManagement.DocumentSeed = AgreementManagement.DocumentSeed(
      id = newId,
      name = d.name,
      prettyName = d.prettyName,
      contentType = d.contentType,
      path = newPath
    )
```